From b7a9b843a86132445c075141e47bb25293f8d636 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Mon, 24 Mar 2025 18:41:13 +0000
Subject: [PATCH] fix setuptools and license

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 3efc6d7..92bfd98 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,11 +1,11 @@
 [build-system]
-requires = ['setuptools>=77.0', 'Cython>=3', "poetry-core>=2.0.0"]
+requires = ['setuptools>=76.0', 'Cython>=3', "poetry-core>=2.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [project]
 name = "habluetooth"
 version = "3.38.0"
-license = "Apache-2.0"
+license = { file = "LICENSE" }
 description = "High availability Bluetooth"
 authors = [{ name = "J. Nick Koston", email = "bluetooth@koston.org" }]
 readme = "README.md"
