From 211df19711686d8b4eea910d0d192eba1e002384 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Mon, 24 Mar 2025 18:41:13 +0000
Subject: [PATCH] fix setuptools

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 6cd8852..234a5c2 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools>=77.0', 'Cython>=3,<3.1', "poetry-core>=2.0.0"]
+requires = ['setuptools>=77.0', 'Cython>=3', "poetry-core>=2.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [project]
