From a9d00dd2c9fbddb2658d010c04eba5898fe51be9 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Thu, 6 Apr 2023 22:01:19 +0200
Subject: [PATCH] Patch pyproject.toml to use upstream version of setuptools
 and wheel

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 5acc91b..8b29e0d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=62.3", "wheel~=0.37.1"]
+requires = ["setuptools>=62.3", "wheel>=0.37.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
