From b0d5c94a121eae8a59b0a8a0766ace04a5353d6d Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Tue, 25 Mar 2025 16:31:37 +0000
Subject: [PATCH] Fix license field

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 00f1009..736dfd1 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 [project]
 name = "bluetooth-data-tools"
 version = "1.26.2"
-license = "Apache-2.0"
+license = { file = "LICENSE" }
 description = "Tools for converting bluetooth data and packets"
 authors = [{ name = "J. Nick Koston", email = "nick@koston.org" }]
 readme = "README.md"
