From 0e4f548f66c507a516a674a90f4d988bbe6fa12d Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Mon, 24 Mar 2025 22:14:40 +0000
Subject: [PATCH] allow older setuptools version

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index caee38b..00f1009 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ['setuptools>=77.0', 'Cython', "poetry-core>=2.0.0"]
+requires = ['setuptools>=76.0', 'Cython', "poetry-core>=2.0.0"]
 build-backend = "poetry.core.masonry.api"
 
 [project]
