From 64622004655b9525af87f74a6732128c085b1faa Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Tue, 4 Apr 2023 00:37:57 +0200
Subject: [PATCH] Fix compilation error for orjson

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 4ff9c03..724cf7b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -83,7 +83,7 @@ debug = false
 incremental = false
 lto = "thin"
 opt-level = 3
-panic = "abort"
+# panic = "abort"
 
 [profile.release.build-override]
 opt-level = 0
