From ebb489f161da241380f6053bd6c1acfe948bf0d7 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sat, 27 Sep 2025 00:55:31 +0200
Subject: [PATCH] use poetry core

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 1c55b54..e61dfee 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -37,5 +37,5 @@ combine_as_imports = "true"
 extra_standard_library = "_socket"
 
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=0.12"]
+build-backend = "poetry.core.masonry.api"
