From fa41f791df85209814b869f7cb72e3590d810097 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Fri, 21 Mar 2025 14:08:43 +0000
Subject: [PATCH] relax setuptools version requirement

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index c4d13eb..23e77e4 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=75.1"]
+requires = ["setuptools>=75.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
