From 844fc844c6e6c4ded5e88f5a4bddc9bfe19f9570 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sun, 1 Sep 2024 19:38:30 +0000
Subject: [PATCH] Make project PEP-517 compliant

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 51 ++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 51 insertions(+)
 create mode 100644 pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
new file mode 100644
index 0000000..207c997
--- /dev/null
+++ b/pyproject.toml
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "pbr"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "aioresponses"
+version = "0.0.0"  # specify the actual version here
+description = "Mock out requests made by ClientSession from aiohttp package"
+readme = "README.rst"
+requires-python = ">=3.7"
+license = { text = "MIT License" }
+authors = [
+    { name = "Pawel Nuckowski", email = "p.nuckowski@gmail.com" },
+]
+keywords = ["aiohttp", "testing", "mocking"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Topic :: Internet :: WWW/HTTP",
+    "Topic :: Software Development :: Testing",
+    "Topic :: Software Development :: Testing :: Mocking",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+
+[project.urls]
+Homepage = "https://github.com/pnuckowski/aioresponses"
+
+[tool.setuptools]
+packages = ["aioresponses"]
+package-data = {"aioresponses" = ["py.typed"]}
+
+[tool.setuptools.exclude-package-data]
+# Exclude files from package distributions
+exclude = ["docs"]
+
+[tool.flake8]
+exclude = ["docs"]
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+
+[tool.pbr]
+# If you use pbr options like versioning, you can specify them here
