From b07370f34342afdd0fae7208aed46e08efb65ec5 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Mon, 10 Apr 2023 23:55:59 +0200
Subject: [PATCH] Change tool version to allow newer version

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 50d335e..1f4e4f6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=68.0", "wheel~=0.40.0"]
+requires = ["setuptools>=67.6.0", "wheel~=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
