From c8133a100c1515943e0d0bd679a0d82410368e64 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Mon, 10 Apr 2023 23:55:59 +0200
Subject: [PATCH] Change tool version to allow newer version

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ebd3566..29190dd 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=62.3", "wheel~=0.37.1"]
+requires = ["setuptools>=62.3", "wheel>=0.37.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
