From 685402da02b8f976f404546e9dff767b10c6005f Mon Sep 17 00:00:00 2001
From: Pascal Bach <pascal.bach@siemens.com>
Date: Thu, 4 May 2017 11:28:44 +0200
Subject: [PATCH] Do not install tinydtls it's only used internally

Upstream-Status: Inappropriate embedded specific

---
 Makefile.in | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index 932919c..349465a 100644
--- a/ext/tinydtls/Makefile.in
+++ b/ext/tinydtls/Makefile.in
@@ -104,13 +104,6 @@ dist:	$(FILES) $(DISTSUBDIRS)
 	tar czf $(package).tar.gz $(DISTDIR)
 
 install:	$(LIB) $(HEADERS) $(SUBDIRS)
-	test -d $(libdir) || mkdir -p $(libdir)
-	test -d $(includedir) || mkdir -p $(includedir)
-	$(install) $(LIB) $(libdir)/
-	$(install) $(HEADERS) $(includedir)/
-	for dir in $(SUBDIRS); do \
-		$(MAKE) -C $$dir install="$(install)" includedir=$(includedir) install; \
-	done
 
 TAGS:	
 	$(ETAGS) -o $@.new $(SOURCES) 
-- 
2.1.4

