From 4580bfa59741d7e84052544f6af39919cad8ffe7 Mon Sep 17 00:00:00 2001
From: Stefano Pagnottelli <stefano.pagnottelli@siralab.com>
Date: Thu, 28 Dec 2017 10:24:55 +0100
Subject: [PATCH] Improve make clean compatibility

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 3f8ca0f..5f944ad 100644
--- a/Makefile
+++ b/Makefile
@@ -78,8 +78,8 @@ clean:
 	@rm -rf coverage
 	@rm -rf build
 	@rm -rf cover
-	@make -C docs clean
-	@python setup.py clean
+	@make -C docs clean SPHINXBUILD=false
+	@python3 setup.py clean
 	@rm -f aiohttp/_frozenlist.html
 	@rm -f aiohttp/_frozenlist.c
 	@rm -f aiohttp/_frozenlist.*.so
-- 
2.7.4

