From fa3de56e9c5f15d2f63f096873497d10fa11bf23 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sun, 6 Oct 2024 17:03:52 +0000
Subject: [PATCH] relax build tool dependencies

Upstream-Status: Pending
Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 6ba4d38..a7eb7ef 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
-    "setuptools~=68.0.0",
-    "wheel~=0.40.0",
+    "setuptools>=68.0.0",
+    "wheel>=0.40.0",
 ]
 build-backend = "setuptools.build_meta"
 
