From 99dfcdddbf9b9931159cb771291111fb4761b81c Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Wed, 26 Mar 2025 21:40:45 +0000
Subject: [PATCH] relax setuptools req

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 6865e37..cd3f650 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
-    "setuptools~=68.0.0",
-    "wheel~=0.40.0",
+    "setuptools>=68.0.0",
+    "wheel>=0.40.0",
 ]
 build-backend = "setuptools.build_meta"
 
