From d53f2528d34b8c427bdba862c4de56e03a96bf4c Mon Sep 17 00:00:00 2001
From: Stefano Pagnottelli <stefano.pagnottelli@siralab.com>
Date: Thu, 28 Dec 2017 10:08:47 +0100
Subject: [PATCH] remove typing, it is already included in python 3.5

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index fe60a15..2ce9c10 100755
--- a/setup.py
+++ b/setup.py
@@ -52,7 +52,6 @@ REQUIRES = [
     'pip>=8.0.3',
     'jinja2>=2.9.6',
     'voluptuous==0.10.5',
-    'typing>=3,<4',
     'aiohttp==2.3.6',   # If updated, check if yarl also needs an update!
     'yarl==0.16.0',
     'async_timeout==2.0.0',
-- 
2.7.4

