From be98dc86327e133c9e606467fef8a29428f9ff80 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Wed, 12 Apr 2023 01:12:53 +0200
Subject: [PATCH] Update pyproject.toml to allow compilation

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 563b0757..8e65b18b 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=62.3", "wheel~=0.37.1"]
+requires = ["setuptools>=62.3", "wheel>=0.37.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
