From 346d61baaaea4c3261642c00bc19d227bd1527ac Mon Sep 17 00:00:00 2001
From: Pascal Bach <pascal.bach@nextrem.ch>
Date: Sun, 12 Feb 2017 12:53:50 +0100
Subject: [PATCH] remove typing, it is already included in python 3.5

---
 setup.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/setup.py b/setup.py
index 0f2c6e6..a4de5d3 100755
--- a/setup.py
+++ b/setup.py
@@ -21,7 +21,6 @@ REQUIRES = [
     'pip>=7.1.0',
     'jinja2>=2.9.5',
     'voluptuous==0.10.5',
-    'typing>=3,<4',
     'aiohttp==2.2.3',
     'async_timeout==1.2.1',
 ]
--
2.1.4
