From bbfc9eea00e036b7181229c84da7befb00b8c2a3 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Wed, 12 Apr 2023 01:12:53 +0200
Subject: [PATCH] Update pyproject.toml to allow compilation

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index fecbea62..ceede209 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=62.3", "wheel~=0.37.1"]
+requires = ["setuptools>=62.3", "wheel>=0.37.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
@@ -36,7 +36,7 @@ dependencies    = [
     # httpcore, anyio, and h11 in gen_requirements_all
     "httpx==0.23.3",
     "home-assistant-bluetooth==1.9.3",
-    "ifaddr==0.1.7",
+    "ifaddr>=0.1.7",
     "jinja2==3.1.2",
     "lru-dict==1.1.8",
     "PyJWT==2.6.0",
