From 64da490e229dc1355021b19326755d0e9be04cef Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Wed, 12 Apr 2023 01:12:53 +0200
Subject: [PATCH] Update pyproject.toml to allow compilation

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 26a9525a..6fb5ad5e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=68.0", "wheel~=0.40.0"]
+requires = ["setuptools>=67.6", "wheel~=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
