From f2edeb31fb1c325dfa10bdc17e04197b93cc4d97 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Tue, 2 Jan 2024 23:32:08 +0000
Subject: [PATCH] Allowed for newer wheel version

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 42670dd..ba5386d 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=68.0", "wheel~=0.40.0"]
+requires = ["setuptools>=68.0", "wheel>=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
