From 8b42ff499cb146d2b7a84ffcbcd04eebcc8da2b5 Mon Sep 17 00:00:00 2001
From: Pascal Bach <pascal.bach@siemens.com>
Date: Tue, 28 Nov 2017 12:18:22 +0100
Subject: [PATCH] Pass configure options from bitbake

Upstream-Status: Inappropriate embedded specific

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 13ac78d..031c099 100644
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@ class prepare_tinydtls(build_ext):
         commands =  [
                     ["autoconf"],
                     ["autoheader"],
-                    ["./configure", "--without-ecc"],
+                    ["./configure", "--without-ecc"] + os.environ.get("CONFIGUREOPTS","").split(),
                     ]
         if not os.path.exists(os.path.join(os.path.dirname(__file__), 'DTLSSocket','tinydtls','dtls.c')):
             run_command(["git", "submodule", "update", "--init"])
-- 
2.11.0

