From e0a12c184463ccd8147e6dbed6034b884b8678c8 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Thu, 15 May 2025 21:58:55 +0000
Subject: [PATCH] relax build deps

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 packaging/pep517_backend/_backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/pep517_backend/_backend.py b/packaging/pep517_backend/_backend.py
index 7588db3..6d5d294 100644
--- a/packaging/pep517_backend/_backend.py
+++ b/packaging/pep517_backend/_backend.py
@@ -379,7 +379,7 @@ def get_requires_for_build_wheel(
     elif sysconfig.get_config_var('Py_GIL_DISABLED'):
         c_ext_build_deps = ['Cython ~= 3.1.0a1']
     else:
-        c_ext_build_deps = ['Cython ~= 3.0.12']
+        c_ext_build_deps = ['Cython >= 3.0.12']
 
     return _setuptools_get_requires_for_build_wheel(
         config_settings=config_settings,
