From 1cfbc6308aeea305ba61a6b8fbf899d57da247bb Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Tue, 27 Aug 2024 22:00:04 +0000
Subject: [PATCH] Correct hatchling build path

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Submitted [https://github.com/zxdavb/evohome-async/pull/22]
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 99e352c..7a73254 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -52,7 +52,7 @@
   sources = ["src"]
 
 [tool.hatch.build.targets.wheel]
-  packages = ["src/evohomeasync", "src/evohomeasync2"]
+  packages = ["evohomeasync", "evohomeasync2"]
 
 [tool.hatch.version]
   path = "src/evohomeasync2/__init__.py"
