From 2ad9920ae050af02ca37f14aae89d75047a386e6 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Mon, 8 Sep 2025 00:16:22 +0200
Subject: [PATCH] relax build req

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 4ee9db3..545d34c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==77.0.3", "wheel==0.45.1"]
+requires = ["setuptools>=77.0.3", "wheel>=0.45.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
