From d98930cbb3f33c845c58cedbf9cb4571def72fa5 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sat, 26 Aug 2023 09:46:12 +0000
Subject: [PATCH] python3-bleak 0.20.1 --> 0.20.2

Upstream-Status: Pending

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index e43408c..0de185e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -42,7 +42,7 @@ dbus-fast = { version = "^1.83.0", markers = "platform_system == 'Linux'" }
 Sphinx = "^5.1.1"
 sphinx-rtd-theme = "^1.0.0"
 
-[tool.poetry.group.lint.dependencies]
+#[tool.poetry.group.lint.dependencies]
 black = "^22.1.0"
 flake8 = "^5.0.0"
 
