From ffa8485caa50b1fb6bfb293e18f54a8d70afe0f3 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sun, 5 May 2024 15:24:46 +0000
Subject: [PATCH] Allow newer version of setuptools

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ccb1c38af5..0b82d4ebf4 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==69.2.0", "wheel~=0.43.0"]
+requires = ["setuptools>=69.2.0", "wheel>=0.43.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
