From feb468400b1a825d587094f0151fbe396ef51f19 Mon Sep 17 00:00:00 2001
From: Tom Geelen <tom.geelen@demcon.com>
Date: Wed, 12 Apr 2023 01:12:53 +0200
Subject: [PATCH] Update pyproject.toml to allow compilation

Signed-off-by: Tom Geelen <tom.geelen@demcon.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index b6206f107f..a9d32ec696 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==68.0.0", "wheel~=0.40.0"]
+requires = ["setuptools>=68.0.0", "wheel>=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
