From fb830ccc1208e3251dfca629005823cc7cb1f517 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Mon, 11 Nov 2024 18:03:19 +0000
Subject: [PATCH] Relax setuptools requirement

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 58a1e8c1659..a70f78ec084 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==75.1.0"]
+requires = ["setuptools>=72.1.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
