From 43f7475c367762a83796f81a5e27ecb839e29481 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Thu, 4 Apr 2024 21:51:46 +0000
Subject: [PATCH] Adjust min version of wheel and setuptools

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 38be44c..fab834e 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==68.0.0", "wheel==0.40.0"]
+requires = ["setuptools>=68.0.0", "wheel>=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
