From 3312d15c8e088a5d8853c8eef0e9423888660414 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Sun, 8 Feb 2026 18:32:25 +0100
Subject: [PATCH] relax build reqs

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
Upstream-Status: Pending
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index fdd17db..646592f 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==80.9.0", "wheel==0.46.1"]
+requires = ["setuptools>=80.9.0", "wheel>=0.46.1"]
 build-backend = "setuptools.build_meta"
 
 [project]
