From 3d0cd2cd73c1e5189ec307e11cb1dbe345a46ea2 Mon Sep 17 00:00:00 2001
From: Tom Geelen <t.f.g.geelen@gmail.com>
Date: Thu, 4 Apr 2024 21:51:46 +0000
Subject: [PATCH] Adjust min version of wheel and setuptools

Signed-off-by: Tom Geelen <t.f.g.geelen@gmail.com>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index e7eb8a6..43392d6 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==68.0.0", "wheel==0.40.0"]
+requires = ["setuptools>=68.0.0", "wheel>=0.40.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
